src/app/address/routes/address/steps.js

```diff
@@ -12,8 +12,19 @@ module.exports = {
   },
   "/prepopulate": {
     controller: prepopulate,
-    next: "search",
     skip: true,
+    next: [
+      {
+        field: "context",
+        value: "international_user",
+        next: "what-country",
```

Comment:
Should this routing now work with the International Address button for the core stubs, I cannot seem to locally?